backend/framework/qlf/dashboard/bokeh/plots/patch/main.py

```diff
@@ -123,3 +136,114 @@ def plot_amp(self, dz, refexp, name="", font_size="1.2vw", description="", wrg=[
                text_font_style="bold", text_font_size="2.5vw", **text_props)
 
         return p
+
+    def plot_amp_status(self, dz, refexp, name="", font_size="1.2vw", description="", nrg=[], wrg=[]):
```

Comment:
I think is possible to reuse the `plot_amp` and make a condition if it contains `nrg` to avoid repeating code unnecessarily.

Reply:
You are right, I will update that